### PR TITLE
SomePythonThings.ZipManager version 4.2.0

### PR DESCRIPTION
**Pull request by marticliment**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28535)

### CODE REVIEW

File: manifests/s/SomePythonThings/ZipManager/4.2.0/SomePythonThings.ZipManager.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: SomePythonThings.ZipManager
+PackageVersion: 4.2.0
```
Comment (ItzLevvie) [inline manifests/s/SomePythonThings/ZipManager/4.2.0/SomePythonThings.ZipManager.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.2
```

File: manifests/s/SomePythonThings/ZipManager/4.2.0/SomePythonThings.ZipManager.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: SomePythonThings.ZipManager
+PackageVersion: 4.2.0
```
Comment (ItzLevvie) [inline manifests/s/SomePythonThings/ZipManager/4.2.0/SomePythonThings.ZipManager.installer.yaml:5]:
```suggestion
PackageVersion: 4.2
```

File: manifests/s/SomePythonThings/ZipManager/4.2.0/SomePythonThings.ZipManager.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: SomePythonThings.ZipManager
+PackageVersion: 4.2.0
```
Comment (ItzLevvie) [inline manifests/s/SomePythonThings/ZipManager/4.2.0/SomePythonThings.ZipManager.yaml:5]:
```suggestion
PackageVersion: 4.2
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image
image

**Comment by martinet101** [comment]
@ItzLevvie  I just removed the .0 from the 4.2.0 as you suggested, and now Azure pipelines don't accepyh the manifest as a valid manifest

**Pull request closed without merging**